en/docs/reference/connectors/pulsar-connector/pulsar-connector-example.md

```diff
@@ -0,0 +1,260 @@
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/AddExpression.png" title="Adding the send parameters." width="700" alt="Configuring publishMessage operation."/>
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/SelectFromPayload.png" title="Adding the send parameters." width="700" alt="Configuring publishMessage operation."/>
+
+    This will open a new pop-up window where you can select the `payload.message` parameter from the request payload. <br/>
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/AddMessageConfig.png" title="Adding the send parameters." width="700" alt="Configuring publishMessage operation."/>
+
+    Next, do the same for the `Key`, `Message Properties`, and `Topic` fields. <br/>
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/PublishMessageConfigPane1.png" title="Adding the send parameters." width="400" alt="PublishMessage operation pane."/>
+  
+    Tick the `Overwrite body` checkbox in the `Output` field to overwrite the message body with the response of the operation. This will replace the sample request payload defined at the top with the output schema of the operation response shown below. <br/>
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/OutputSchema.png" title="publishmessage operation output schema." width="400" alt="publishmessage operation output schema."/>
+
+13. Add a Log mediator to log the response of the `publishMessage` operation. To do this, click on the `+` icon after the `publishMessage` operation and select the `Log` mediator from the **Mediator Palette**.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/AddLogMediator1.png" title="Adding Log Mediator" width="700" alt="Adding Log Mediator"/>
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/AddLogMediator2.png" title="Adding Log Mediator" width="500" alt="Adding Log Mediator"/>
+
+
+14. Add the [Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/) to respond to the response of the `publishMessage` operation as shown below.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/AddRespondMediator.png" title="Adding the respond mediator." width="900" alt="Adding the respond mediator."/>
+
+## Export integration logic as a carbon application
+To export the project, refer to the [build and export the carbon application]({{base_path}}/develop/deploy-artifacts/#build-and-export-the-carbon-application) guide.
+
+## Get the project
+
+You can download the ZIP file and extract the contents to get the project code.
+
+<a href="{{base_path}}/assets/attachments/connectors/PulsarProducer.zip">
+    <img src="{{base_path}}/assets/img/integrate/connectors/pulsar/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+## Deployment
+
+To deploy and run the project, refer to the [build and run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.
+
+## Test
```

Comment:
Let's have the steps to try this using the built-in swagger tryout in our extension instead of using curl.